scrapy/core/scheduler.py

```diff
@@ -84,11 +84,16 @@ def _dqpush(self, request):
         try:
             reqd = request_to_dict(request, self.spider)
             self.dqs.push(reqd, -request.priority)
-        except ValueError as e: # non serializable request
+        except ValueError as e:  # non serializable request
             if self.logunser:
-                logger.error("Unable to serialize request: %(request)s - reason: %(reason)s",
-                             {'request': request, 'reason': e},
+                msg = ("Unable to serialize request: %(request)s - reason:"
+                       " %(reason)s - no more unserializable requests will be"
+                       " logged (stats being collected)")
```

Comment:
`(stats being collected)` could be changed to something like `(see 'scheduler/unserializable' stats counter)`